docs/source/format/Glossary.rst

```diff
@@ -197,6 +197,10 @@ Glossary
 
        Not part of the columnar format; this term is specific to
        certain language implementations of Arrow (primarily C++ and
-       its bindings).
+       its bindings and Go).
```

Comment:
Let's be less specific perhaps, and say "for example C++ and its bindings"?

Reply:
I wrote "(for example C++ and its bindings, and Go).", since I felt that Go was a useful detail. Is that acceptable to you?

Reply:
Definitely, thanks!